test/cljs/status_im/test/utils/money.cljs

```diff
@@ -0,0 +1,11 @@
+(ns status-im.test.utils.money
+  (:require [cljs.test :refer-macros [deftest is]]
+            [status-im.utils.money :as money]))
+
+(deftest bignumber
+  (is (= (str (money/wei->ether 111122223333444000))
+         "0.111122223333444")))
+
+(deftest bignumber-2
```

Comment:
A better error message would be generated by having a more descriptive test name (say `wei->ether`) then relying on `testing` macro to distinguish scenarios. A single test would be sufficient then.